src/virtualenv/create/via_global_ref/builtin/python2/site.py

```diff
@@ -114,6 +108,45 @@ def abs_path(value):
     return value
 
 
+def translate_directory(directory):
```

Comment:
this should be after ``rewrite_standard_library_sys_path`` to ordered top-down by the occurrence